docs/CONTRIBUTING.md

```diff
@@ -11,7 +19,7 @@ Coder requires Go 1.18+, Node 14+, and GNU Make.
 
 Use the following `make` commands and scripts in development:
 
-- `make dev` runs the frontend and backend development server
+- `./scripts/develop.sh` runs the frontend and backend development server
```

Comment:
@johnstcn since `dev` is deprecated in `Makefile` should I delete it there?

Reply:
Yes, good catch!

Reply:
Cool! I'll it in this PR unless anyone has any objections.